app/app.go

```diff
@@ -731,10 +784,12 @@ func NewIxoAnteHandler(app *IxoApp, encodingConfig params.EncodingConfig) sdk.An
 		app.AccountKeeper, app.BankKeeper, app.DidKeeper,
 		encodingConfig.TxConfig.SignModeHandler(), projectPubKeyGetter)
 
+	// TODO: Routing https://docs.cosmos.network/v0.44/building-modules/msg-services.html#amino-legacymsgs
 	return func(ctx sdk.Context, tx sdk.Tx, simulate bool) (_ sdk.Context, err error) {
 		// Route message based on ixo module router key
 		// Otherwise, route to Cosmos ante handler
-		msg := tx.GetMsgs()[0]
+		msg := tx.GetMsgs()[0].(legacytx.LegacyMsg)
+
```

Comment:
@ig-shaun @demondayza @steveixo 
Okay so the original message interface was deprecated in the Cosmos-SDK with one that ties into the proto message interface. As mentioned to @demondayza we can use this legacy message for now but would need ot upgrade our modules first to use protobuf correctly first.

Reply:
Is this implementation still functional with the legacy message in place? If so then we can resolve this comment and plan for a full protobuf migration for the modules but that will need to include some backwards compatibility.

Reply:
From my understanding this is still functional.